tests/specs/unit/modules/apiMe.js

```diff
@@ -149,6 +149,20 @@ define(['./helper'], function(helper) {
 				}
 			},
 			{
+				network: 'vk',
+				expect: {
+					id: 434460,
+					name: 'Юрий Опарин',
+					thumbnail: 'http://cs304605.vk.me/u434460/d_1acca7c0.jpg'
+				},
+				errorExpect: {
+					error: {
+						error_code: 10,
```

Comment:
In your module/vk.js could you format the errors, so they are consistent, i.e. an `error` property with an Object of the following format...

```
error: {
      code: 10,
      message: 'Internal server error: could not get application'
}
```